packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -312,9 +312,9 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
 
     // Only check for hovering if there is a mouse connected.
     if (_mouseIsConnected) {
-      result = Listener(
-        onPointerEnter: (PointerEnterEvent event) => _showTooltip(),
-        onPointerExit: (PointerExitEvent event) => _hideTooltip(),
+      result = MouseRegion(
+        onEnter: (PointerEnterEvent event) => _showTooltip(),
+        onExit: (PointerExitEvent event) => _hideTooltip(),
```

Comment:
why do we use methods in the previous two files and closures here? we should be consistent.

Reply:
I agree, but I'm leaning towards making as few changes in other files as possible since it's a breaking PR.

Reply:
not sure why it being a breaking PR matters here?

But if you want to change it in another PR that's fine. :-)